src/crypto/tests/CHIPCryptoPALTest.cpp

```diff
@@ -902,6 +902,42 @@ static void TestPBKDF2_SHA256_TestVectors(nlTestSuite * inSuite, void * inContex
     NL_TEST_ASSERT(inSuite, numOfTestsRan > 0);
 }
 
+static void TestP256_Keygen(nlTestSuite * inSuite, void * inContext)
```

Comment:
I don't see any code that verifies these calls don't overflow the passed in buffers

Reply:
There's one test for the overflow: https://github.com/project-chip/connectedhomeip/pull/2467/files#diff-d646ceb2b2d4e7ad594340646baa5f1bR920